sycl/test/lit.cfg.py

```diff
@@ -166,6 +166,10 @@ def getDeviceCount(device_type):
     if platform.system() == "Linux":
         gpu_run_on_linux_substitute = "env SYCL_DEVICE_TYPE=GPU SYCL_BE={SYCL_BE} ".format(SYCL_BE=backend)
         gpu_check_on_linux_substitute = "| FileCheck %s"
+    # ESIMD-specific setup. Requires OpenCL for now.
```

Comment:
It is better to check that opencl feature is present as it is requirement

Reply:
For now I want the tests to fail when OpenCL feature is not present on GPU, and be able to SYCL_BE=PI_OPENCL